### DISCUSSION TITLE
Generating images?

### DISCUSSION THREAD

**Discussion by kbalicki** [mckaywrigley/chatbot-ui#1824] [Help]
Hello,

Every time I try to generate some image I get reply like "I'm sorry, but I can't generate or display images directly..."
I have tried different models, settings...

The same prompts work perfectly fine at https://chatgpt.com/

Any ideas what could be wrong ?

**Comment by bernie43** [comment]
ChatGPT has Dall-E integration for image generation.
Chatbot UI only uses plain GPT, no Dall-E, so it can‘t generate any images.

**Reply by kbalicki** [reply]
Tank you for te information, tat makes sense.
